### ISSUE TITLE
bpipe can not support division or divide function from groovy  ？

### ISSUE THREAD

**Issue by WinterLi1993** [ssadedin/bpipe#192]
Hi，
 I tried ,
    def x = 26
   def y = 10
   def resultDiv = x.div(y)  // or x / y
   but Bpipe can not support these codes from a groovy script !

**Comment by wljlinksly** [comment]
Of course, Bpipe support groovy script of various mathematical operations, your problem may lay in the way you implement your code or the way you run your script.

**Comment by ssadedin** [comment]
Hi,

Can you say what you mean by it not supporting it? I made a script like this:

```groovy
def x = 26
def y = 10
def resultDiv = x.div(y) 

println "Result = $resultDiv"
```

Then I ran it:

```bash
$ bpipe run test.groovy 
Result = 2.6
```

So I think it printed the correct result?

**Comment by WinterLi1993** [comment]
it's right, thank you
